### PR TITLE
bones

### PR DESCRIPTION
**Pull request by aurorahager**
Very rough module view for students with hard coded data.
only looks okay on ipad view currently.
added ng-youtube-embed to vendors.

### CODE REVIEW

**Review by ericbfriday — DISMISSED** [review]
We have a lot of /demo bloat in the vendors file that is unneeded. Below are some removes and notes. 
Remove ng-youtube-embed/README.md
Remove ng-youtube-embed/demo/app.js
Remove ng-youtube-embed/demo/index.html
Remove ng-youtube-embed/demo/styles.css
Remove /ng-youtube-embed/icon.png
/views/templates/lesson.html -> Do a song with less explicit lyrics. Let’s keep our GitHub repo school appropriate.

File: server/public/styles/style.css [hunk 1]
```diff
@@ -80,3 +78,19 @@ li a:hover {
   text-decoration: underline;
   color: #eee;
 }
+
+/* Student module */
+
+/* increase size of info in module */
+.sizeMd {
```
Comment (mikeekert) [inline server/public/styles/style.css]: Can we swap the pixel sizes to rem's (this allows us to adjust the entire site's font sizes universally if we want to scale them up/down all together).

**Review by carlabrady — DISMISSED** [review]
looks good

File: server/public/views/index.html [hunk 2]
```diff
@@ -3,26 +3,33 @@
...
+        <!-- angular material -->
         <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.4/angular-aria.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.4/angular-animate.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-messages/1.6.4/angular-messages.min.js"></script>
         <script src="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.4/angular-material.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-sanitize/1.6.6/angular-sanitize.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/md-data-table/2.2.0/md-data-table.min.js "></script>
+        <script src="vendors/ng-youtube-embed/build/ng-youtube-embed.min.js"></script>
         <script src="vendors/angular-jk/jk-carousel.js"></script>
+        <script src="vendors/angular-jk/jk-carousel.js"></script>
```
Comment (ericbfriday) [inline server/public/views/index.html]: Sourcing carousel twice. Remove one.

File: server/public/views/index.html [hunk 3]
```diff
@@ -3,26 +3,33 @@
...
+        <script src="vendors/ng-youtube-embed/build/ng-youtube-embed.min.js"></script>
```
Comment (ericbfriday) [inline server/public/views/index.html]: Sourcing youtube-embed twice. Remove one.

**Review by ericbfriday — CHANGES_REQUESTED** [review]

**Review by ericbfriday — APPROVED** [review]
Second Level Review Approved. Merging.

**Pull request merged**